config/sync/taxonomy.vocabulary.opportunity_tag_1.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 6bd22067-d9e5-43ac-9206-6c511941d713
```

Comment:
I would like to find a better solution than 10 separate taxonomies. It makes is very difficult to do much with it and it's very difficult for content editors to know which taxonomy they need to edit.
My idea is to use a single vocabulary that is multiple levels. The top most level is the "Label" and the children will be treated like the field values. I'll have to look to see if there is any contib module for the form UI, but I could probably build one that could do that as well.

Reply:
@buttonwillowsix what do you think about this behavior? I've got an approach where the field form is built by the taxonomy hierarchy. The parent terms are treated like the field labels, and then a user can structure their content as they see fit. If they need 2 categories, they just create 2 parent terms. If they need 20, they create 20 parent terms. There's really no limitation to the number of vocabularies with this approach.
![image](https://github.com/user-attachments/assets/8b7e9094-79af-4533-ad28-f9b7d5c9b4f4)

Reply:
@pookmish. I love this approach. MUCH better. 

Not that this should stop us, but do you think it would be possible to migrate cardinal service vocabs (which is what is driving this insane list) into that model?

So, to accept this PR, would I just trim it down to a single "Tags" vocab? I will make that change.

Reply:
@pookmish I trimmed this down. Please have a look!